tests/Doctrine/Tests/ORM/Query/SelectSqlGenerationTest.php

```diff
@@ -2242,6 +2259,43 @@ public function testHavingSupportResultVariableInAggregateFunction()
             'SELECT COUNT(c0_.name) AS sclr_0 FROM cms_users c0_ HAVING sclr_0 IS NULL'
         );
     }
+
+    /**
+     * @group DDC-3701
+     */
+    public function testFROMClauseAcceptsFQCNWithLeadingBackslash()
+    {
+        $this->_em->createQuery(
```

Comment:
This test lacks an assertion. If you are just verifying that valid SQL is generated, then `$this->assertInternalType('string'), $sql)` is enough